src/families/bitcoin/wallet-btc/utils.ts

```diff
@@ -7,6 +7,7 @@ import { Currency, ICrypto } from "./crypto/types";
 import cryptoFactory from "./crypto/factory";
 import { fallbackValidateAddress } from "./crypto/base";
 import { UnsupportedDerivation } from "../../../errors";
+import varuint from "varuint-bitcoin";
```

Comment:
`varuint-bitcoin` isn't added in _package.json_ can you please add it + update the _dependencies.md_ file?

Reply:
fixed